pkg/mcs/resource_manager/server/manager.go

```diff
@@ -37,6 +37,9 @@ const (
 	defaultConsumptionChanSize = 1024
 	metricsCleanupInterval     = time.Minute
 	metricsCleanupTimeout      = 20 * time.Minute
+
+	defaultGroupName = "default"
```

Comment:
I suggest adding a `reserved` to indicate it's an internal use resource group name, otherwise, it may be regarded as some default name setting.

```suggestion
	reservedDefaultGroupName = "default"
```